gatsby-node.js

```diff
@@ -46,7 +48,7 @@ const createArticles = async (createPage) => {
   const newsResp = await axios.get(`${BASE_URL}https://www.thedp.com/section/news.json`)
   newsResp.data.articles.forEach(article => {
     const { authors } = article
-    authors.forEach(async author => await createAutors(createPage, author.slug))
+    authors.forEach(async author => await createAuthors(createPage, author.slug))
```

Comment:
prefer destructuring on author.slug, so const { authors: { slug} } = article

Reply:
also, can you call promises in a forEach loop like that? Shouldn't you use Promise.all()? Right now these promises run sequentially, we can run them in parallel instead